packages/paste-website/src/pages/form-elements/textarea/index.mdx

```diff
@@ -0,0 +1,598 @@
+
+### Errors
+
+Use inline error text to explain how to fix an error.
+
+Ideally, help text should have enough information to help users prevent errors. If help text exists and you need to show an error, the error text should replace and repeat the help text until the error has been resolved.
+
+Error text should:
+
+- Be actionable. Explain how to fix an error and if reasonable, why it happened so that it might also be prevented in the future.
+- Be concise and simple, maybe even more than normal. Avoid jargon. Try to keep error text to 2 sentences or fewer.
+- Use the passive voice for textarea errors to avoid placing blame on the user. For example, "A friendly name is required."
+- Use the active voice for system errors. For example, "Our systems are currently down. Please contact our support team."
+
+### Optional compositional elements
+
+- **Prefix/suffix** — Use a prefix or suffix to help users format their textarea and to provide more context about the value a user is entering. For example, you could prefix or suffix a field with a currency symbol, or use suffix to append a character counter. Make sure to consider internationalization when using prefixes or suffixes since formatting may differ across languages. For example, currency symbols are placed on the left in American English, while they're placed on the right in French. Don't use prefix/suffix text as a replacement for a label.
+- **Icon** — Use an icon if you need to give the user additional controls for the field. For example, use an icon to clear a field on click, removing the need for users to manually delete their entered value. Place icon buttons that trigger an action on the right side of the field. If you need 2 actions on a field (e.g., popover trigger and clear field), place the icon button that affects the field value on the right, and the other icon on the left.
+- **Placeholder** — Use a placeholder when you want to give an example of the type of data a user should enter, mainly to help with recall. For example, when asking for links to social media profiles, you could give examples of a few social media sites. Placeholders should be used in addition to the label and not as a replacement since they'll disappear when a user begins entering their own value. If you need to show supporting instructions, use help text instead.
+
+## When to use an textarea
+
+Use a textarea when users are expected to enter text that exceeds a single line, usually longer than a sentence.
+
+<DoDont>
+  <Do title="Do" body="Use a textarea to encourage longer text entry." center>
+    <Box width="100%" padding="space60">
+      <FormLabel htmlFor="long_do">Tell us your life story</FormLabel>
+      <FormTextArea id="long_do" name="long_do"/>
+    </Box>
+  </Do>
+  <Dont
+    title="Don't"
+    body="Don't use a textarea when text entry is expected to be short since it could confuse users. Use an [input]() instead."
```

Comment:
I think the `[input]()` is intended to be a link and we can't achieve that with the body being a prop. You'll have to remove those for now.

Reply:
Removed.